src/main/java/com/neighborhoodAssurance/NAController.java

```diff
@@ -10,30 +10,31 @@
 public class NAController {
 	
 	//private ServiceStub
+	private final String HOME_PATH = "/index";
 
-	@RequestMapping(value="/index", method=RequestMethod.GET) 
+	@RequestMapping(value=HOME_PATH, method=RequestMethod.GET) 
 	public String index()
 	{
-		return "index";
+		return HOME_PATH.replace("/", "");
 		
 	}
-	@RequestMapping(value="/index", method=RequestMethod.GET, headers= {"content-type=text/json"}) 
+	@RequestMapping(value=HOME_PATH, method=RequestMethod.GET, headers= {"content-type=text/json"}) 
 	@ResponseBody
 	public String readJSON() 
 	{
 		/*
 		 * DTO Information
 		 */
-		return "index"; //return DTO
+		return HOME_PATH.replace("/", ""); //return DTO
 	}
-	/*@RequestMapping(value="/index", method=RequestMethod.GET)
+	/*@RequestMapping(value=HOME_PATH, method=RequestMethod.GET)
 	public String readSearchValue(@RequestParam(value="searchTerm", required=true)String searchTerm) //May need defaultValue="
 	{  
 		/*
 		 * DTO Information?
 		 
 		
-		return "index";
+		return HOME_PATH.replace("/", "");
 		}*/
 	 

```

Comment:
This is commented out code, so I'd remove it all together, instead of maintain it behind the comments.

 Instead of commenting out code, my suggestion is:
1) Use feature branches in Git to keep experimental work separate from master.
2) If you're removing something that used to be there, just remove it entirely.  You can always go back in history to see what used to be there.